cylc/flow/scripts/lint.py

```diff
@@ -14,19 +14,18 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Cylc configuration linter.
+
+Checks code style, deprecated syntax and other issues."""
```

Comment:
The docstring is functional for `cylc help all`, just need to give it two lines to follow the pattern of other commands.